docs/pages/versions/unversioned/sdk/device.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Device
+---
+
+Provide information of devices on the application.
+
+## Installation
+
+This API is pre-installed in [managed](../../introduction/managed-vs-bare/#managed-workflow) apps. It is not yet available for [bare](../../introduction/managed-vs-bare/#bare-workflow) React Native apps.
+
+## API
+
+```js
+import { Device } from 'expo';
+```
+
+### Constants
+
+#### `Device.brand: string`
```

Comment:
Would it be possible to make this list more concise (as a list for example?), maybe with some examples? So that users know what's the difference between `brand`, `manafacturer` and `model`